includes/class-admin.php

```diff
@@ -469,4 +477,39 @@ public static function handle_bulk_request( $sendback, $action, $users ) {
 
 		return $sendback;
 	}
+
+	/**
+	 * Add ActivityPub infos to the dashboard glance items
+	 *
+	 * @param array $items The existing glance items.
+	 *
+	 * @return array The extended glance items.
+	 */
+	public static function dashboard_glance_items( $items ) {
+		\add_filter( 'number_format_i18n', '\Activitypub\custom_large_numbers', 10, 3 );
+
+		if ( ! is_user_disabled( get_current_user_id() ) ) {
+			$items['activitypub'] = sprintf(
+				'<a class="activitypub-followers" href="%s" title="%s">%s %s</a>',
+				admin_url( 'users.php?page=activitypub-followers-list' ),
+				esc_attr__( 'Your followers', 'activitypub' ),
+				\number_format_i18n( count_followers( get_current_user_id() ) ),
+				__( 'Followers', 'activitypub' )
+			);
```

Comment:
Instead of splitting this into 2 strings, would it be worth building that string before, so it can better support i18n and different types of numbers, with `_n`? Something like this?

```suggestion
			$follower_count = sprintf(
				_n(
					'%s follower',
					'%s followers',
					count_followers( get_current_user_id() ),
					'activitypub'
				),
				number_format_i18n( count_followers( get_current_user_id() ) )
			);
			$items['activitypub'] = sprintf(
				'<a class="activitypub-followers" href="%1$s" title="%2$s">%3$s</a>',
				esc_url( admin_url( 'users.php?page=activitypub-followers-list' ) ),
				esc_attr__( 'Your followers', 'activitypub' ),
				esc_html( $follower_count )
			);
```